kvdb/config.go

```diff
@@ -35,7 +35,7 @@ const (
 
 // BoltConfig holds bolt configuration.
 type BoltConfig struct {
-	NoFreelistSync bool `long:"nofreelistsync" description:"Whether the databases used within lnd should sync their freelist to disk. This is set to true by default, meaning we don't sync the free-list resulting in imporved memory performance during operation, but with an increase in startup time."`
+	NoFreelistSync bool `long:"nofreelistsync" description:"Whether the databases used within lnd should sync their freelist to disk. This is set to true by default, meaning we don't sync the free-list resulting in improved memory performance during operation, but with an increase in startup time."`
```

Comment:
This should cause a change in the top level `sample-lnd.conf` file as well I think?

Reply:
I found that `config.go` and `sample-lnd.conf` were not exactly the same, and there is no typo error in `sample-lnd.conf`